blankly/utils/utils.py

```diff
@@ -126,15 +349,30 @@ def __compare_dicts(self, default_settings, user_settings):
                     warning_string = "\"" + str(k) + "\" not specified in preferences, defaulting to: \"" + str(v) + \
                                      "\""
                     info_print(warning_string)
-                    user_settings[k] = v
+                    user_settings[k] = v.default
```

Comment:
noticed this mistake earlier, thought corrected it prior to pushing. At this point, v is still a dict, so it has no method "default". Will correct and push the changes